jicoco-config/src/main/kotlin/org/jitsi/config/TypesafeConfigSource.kt

```diff
@@ -46,7 +46,23 @@ class TypesafeConfigSource(
             typeOf<Boolean>() -> wrap { key -> config.getBoolean(key) }
             typeOf<Int>() -> wrap { key -> config.getInt(key) }
             typeOf<Long>() -> wrap { key -> config.getLong(key) }
-            typeOf<Double>() -> wrap { key -> config.getDouble(key) }
+            // Support expressions such as "5%"
+            typeOf<Double>() -> wrap { key ->
```

Comment:
Wondering if we want to move this into a `doubleGetter` helper function to keep this block a bit neater.